industrial_ci/src/tests/ros_prerelease.sh

```diff
@@ -25,12 +25,14 @@ function setup_environment() {
     WORKSPACE=$(mktemp -d)
     echo "WORKSPACE: $WORKSPACE"
 
-    if [ -e /var/run/docker.sock ]; then
+    if [ -n "$DOCKER_PORT" ]; then
```

Comment:
Where is `DOCKER_PORT` passed from? If this is an input from user then could you add this to the variable section?

Reply:
I think this is very specific to Gitlab and their services.
It looks like a Docker link, because of the other environment variables with only parts of the URI.

The (regular) users shold better not pass it directly without a special reason.